style.css

```diff
@@ -1,4 +1,100 @@
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  padding: 0;
+}
+
+header {
+  background-color: #3c3c3c;
+}
+
+/* removed bullet point */
+ul {
+  list-style-type: none;
+}
+
+@media only screen and (min-width: 375px) {
+  /* header section */
+  .header_1 {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 23px;
+  }
+
+  .navbar {
+    display: flex;
+  }
+
+  /* aligned th image to left */
+  .img2 {
+    margin-left: auto;
+  }
+
+  /* main section css */
+  .description_section {
+    background-color: #1c1a19;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    height: 590px;
+    align-content: center;
+    position: relative;
+  }
+
+  .h1_heading {
+    font-size: 3.5rem;
+    color: #ff6b00;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 0 0 10px 16px;
+    width: fit-content;
+  }
+
+  .h3_heading {
+    color: white;
+    font-size: 1.5rem;
+    font-style: normal;
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+    margin: 0 0 10px 16px;
+  }
+
+  .paragraph {
+    font-size: 1rem;
+    font-family: 'Poppins', sans-serif;
+    word-wrap: break-word;
+    color: white;
+    margin: 0 0 0 16px;
+    width: 375px;
```

Comment:
- [ ] I suggest correcting the margin and removing the width of the paragraph to be similar to the Figma design 👇🏽 

```css
.paragraph {
  margin: 0 16px;
}
```

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/60d7ca37-c42c-4d6a-a4f5-4ac08316675a) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/750bb8e7-39ff-4169-a2dc-753249797fa6) |

Reply:
@franclobo Done! should I merge it now!